leapp/reporting/__init__.py

```diff
@@ -0,0 +1,193 @@
+from leapp.models import Model, fields
+from leapp.topics import Topic
+
+
+class ReportTopic(Topic):
+    name = 'report_topic'
```

Comment:
do not forget to move this into the separate file in the `leapp/topics` directory.